applications/trilinos_application/python_scripts/trilinos_contact_strategy.py

```diff
@@ -222,7 +222,7 @@ def InitializeSolutionStep(self, reform_dofs):
             # reorder the list of degrees of freedom to identify fixity and system size
```

Comment:
+1, this is very old and I did it adhoc for tunneling problem with a lot of balancing problem.